code/modules/gear_presets/uscm_co.dm

```diff
@@ -0,0 +1,458 @@
+		WEAR_BODY = /obj/item/clothing/under/marine/officer/command,
+		WEAR_JACKET = /obj/item/clothing/suit/storage/jacket/marine/dress/officer/patchless,
+		WEAR_FEET = /obj/item/clothing/shoes/white,
+		WEAR_HANDS = /obj/item/clothing/gloves/white,
+	)
+
+/datum/equipment_preset/uscm_co/commander/council/medical
+
+	name = "Shipside - CO - Medical - COUNCIL"
+	idtype = /obj/item/card/id/gold/council
+	paygrades = list(PAY_SHORT_MO5 = JOB_PLAYTIME_TIER_0)
+	equipment_to_spawn = list(
+		WEAR_HEAD =	/obj/item/clothing/head/beret/marine/commander/dress ,
+		WEAR_L_EAR = /obj/item/device/radio/headset/almayer/mcom/cdrcom,
+		WEAR_BODY = /obj/item/clothing/under/marine/officer/command,
+		WEAR_JACKET = /obj/item/clothing/suit/storage/jacket/marine/dress/officer/patchless,
+		WEAR_FEET = /obj/item/clothing/shoes/white,
+		WEAR_HANDS = /obj/item/clothing/gloves/white,
+	)
+
+/datum/equipment_preset/uscm_co/commander/council/plus/medical
+
+	name = "Shipside - CO - Medical - SENATOR"
+	idtype = /obj/item/card/id/general
+	paygrades = list(PAY_SHORT_MO6 = JOB_PLAYTIME_TIER_0)
+	equipment_to_spawn = list(
+		WEAR_HEAD =	/obj/item/clothing/head/beret/marine/commander/dress ,
+		WEAR_L_EAR = /obj/item/device/radio/headset/almayer/mcom/cdrcom,
+		WEAR_BODY = /obj/item/clothing/under/marine/officer/command,
+		WEAR_JACKET = /obj/item/clothing/suit/storage/jacket/marine/dress/officer/patchless,
+		WEAR_FEET = /obj/item/clothing/shoes/white,
+		WEAR_HANDS = /obj/item/clothing/gloves/white,
+	)
+
+/datum/equipment_preset/uscm_co/commander/aviation
+
+	name = "Shipside - CO - Aviation"
+	equipment_to_spawn = list(
+		WEAR_L_EAR = /obj/item/device/radio/headset/almayer/mcom/cdrcom,
+		WEAR_BODY = /obj/item/clothing/under/marine/officer/formal/turtleneck,
+		WEAR_L_STORE = /obj/item/storage/pouch/pistol/command,
+		WEAR_R_STORE = /obj/item/storage/pouch/general/large,
+		WEAR_FEET =	/obj/item/clothing/shoes/marine/knife,
+		WEAR_HANDS = /obj/item/clothing/gloves/black,
+		WEAR_HEAD = /obj/item/clothing/head/beret/marine/commander/black,
+		WEAR_JACKET = /obj/item/clothing/suit/storage/jacket/marine/pilot,
+		WEAR_EYES = /obj/item/clothing/glasses/sunglasses/aviator,
+	)
+
+/datum/equipment_preset/uscm_co/commander/council/aviation
+
+	name = "Shipside - CO - Aviation - COUNCIL"
+	idtype = /obj/item/card/id/gold/council
+	paygrades = list(PAY_SHORT_MO5 = JOB_PLAYTIME_TIER_0)
+	equipment_to_spawn = list(
+		WEAR_L_EAR = /obj/item/device/radio/headset/almayer/mcom/cdrcom,
+		WEAR_BODY = /obj/item/clothing/under/marine/officer/formal/turtleneck,
+		WEAR_L_STORE = /obj/item/storage/pouch/pistol/command,
+		WEAR_R_STORE = /obj/item/storage/pouch/general/large,
+		WEAR_FEET =	/obj/item/clothing/shoes/marine/knife,
+		WEAR_HANDS = /obj/item/clothing/gloves/black,
+		WEAR_HEAD = /obj/item/clothing/head/beret/marine/commander/black,
+		WEAR_JACKET = /obj/item/clothing/suit/storage/jacket/marine/pilot,
+		WEAR_EYES = /obj/item/clothing/glasses/sunglasses/aviator,
+	)
```

Comment:
You're not changing anything between council and non-council gear presets, so you don't need to redefine them.